test/blackbox/ddsrouter_core/dds/WAN/DDSTestWAN.cpp

```diff
@@ -0,0 +1,200 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <condition_variable>
+#include <mutex>
+#include <thread>
+
+#include <gtest_aux.hpp>
+#include <gtest/gtest.h>
+#include <TestLogHandler.hpp>
+
+#include <ddsrouter/core/DDSRouter.hpp>
+#include <ddsrouter/types/Log.hpp>
+#include <ddsrouter/types/RawConfiguration.hpp>
+
+#include "../types/Common.hpp"
```

Comment:
Dear reviewer, would you know how to avoid using relative path here?

Reply:
No. My only suggestion would be to use `include_directories` in CMakeLists.txt